app/src/main/java/dev/ukanth/iconmgr/IconAdapter.java

```diff
@@ -62,8 +64,8 @@ public IconPackViewHolder(View view) {
                     IconAttr attr = new Gson().fromJson(currentItem.getAdditional(), IconAttr.class);
                     attr.setFavorite(!attr.isFavorite());
                     currentItem.setAdditional(new Gson().toJson(attr).toString());
-                    App app = ((App) ctx.getApplicationContext());
-                    app.getDaoSession().getIPObjDao().update(currentItem);
+                    IPObjDatabase db = IPObjDatabase.getInstance(ctx.getApplicationContext());
```

Comment:
This should be removed or handled in new way ?

Reply:
Sorry, I forgot about this commented out code.